frontend/src/container/Timeline/styles.ts

```diff
@@ -0,0 +1,19 @@
+import styled, { css } from 'styled-components';
+
+const timelineContainer = css`
+	flex: 1;
+	overflow: visible;
+`;
+
+export const styles = {
+	timelineContainer,
+};
+export const Svg = styled.svg`
+	overflow: visible !important;
```

Comment:
do we really need `!important` ?

Reply:
yes, bcoz something is overriding it somewhere so not able to make this work without this

Reply:
ok